posthog/urls.py

```diff
@@ -92,6 +93,32 @@ def setup_admin(request):
         })
         return redirect('/')
 
+def social_create_user(strategy, details, backend, user=None, *args, **kwargs):
```

Comment:
extended the standard implementation of this function from the source. includes team addition now and so analytics for new users can be logged

Reply:
when we need to implement social auth more thoroughly (through setup_admin, etc.) we can place logic here

Reply:
Great! Would be good to test this, either e2e (see `test_signup.py` for an example) or just this function.